app/models/knight.rb

```diff
@@ -11,22 +11,22 @@ class Knight < GamePiece
   #       end
   #   end
 
-    def valid_move?(next_x, next_y)
-      # these values should be converted to integers before being passed in, if it is necessary
-			n_x = next_x.to_i
-	    n_y = next_y.to_i
+  def valid_move?(next_x, next_y)
+    # these values should be converted to integers before being passed in, if it is necessary
+    n_x = next_x.to_i
+    n_y = next_y.to_i
 
-      #return false if piece doen't move
-			# Not needed anymore; current location not available as a move (Sherilyn)
-      # return false if self.x == n_x && self.y == n_y
+    #return false if piece doen't move
+    # Not needed anymore; current location not available as a move (Sherilyn)
+    # return false if self.x == n_x && self.y == n_y
 
-      # define the rule of knight
-      if (self.x - n_x).abs + (self.y - n_y).abs == 3
-				self.move_piece(next_x, next_y)
-        return true
-      else
-        return false
-      end
+    # define the rule of knight
+    if (self.x - n_x).abs + (self.y - n_y).abs == 3
+      self.move_piece(next_x, next_y)
```

Comment:
I didn't mean to touch knight.rb (see my comment above).  I was playing something in knight.rb before.  I didn't want to push it so I deleted the file and tried to match what we have in master branch.  This knight.rb is, (and should be) the same as in master branch, except spacing.   My editor automatically changed the spacing.  Sorry for the confusion.